ocaml/typing/oprint.ml

```diff
@@ -645,6 +645,11 @@ and print_simple_out_module_type ppf =
   | Omty_alias id -> fprintf ppf "(module %a)" print_ident id
   | Omty_functor _ as non_simple ->
      fprintf ppf "(%a)" print_out_module_type non_simple
+  | Omty_strengthen (mty, id, unaliasable) ->
+     fprintf ppf "(%a with %a%s)"
+       print_simple_out_module_type mty
+       print_ident id
+       (if unaliasable then " [@ unaliasable]" else "")
```

Comment:
This how @lpw25 and I agreed how to print this. It's an extremely rare case and @lpw25 suggested to print it like this and we can always add support for the attribute if it ever becomes necessary.

Reply:
That seems reasonable.  I think there should be at least one example in the testsuite that triggers this, though.  And probably we should add it to the `builtin_attrs` table in `parsing/builtin_attributes.ml` so that if someone ever does try to write it on the basis of this output, they at least get a warning that it's unhandled.

Reply:
Done.

Reply:
Yep - thanks!